### ISSUE TITLE
Cannot use WebSocket Hibernation API

### ISSUE THREAD

**Issue by broccolihighkicks** [cloudflare/workerd#736]
```
workerd = 1.20230518.0
wrangler = 3.0.1
```

When using the example code from here: https://github.com/cloudflare/workers-chat-demo/pull/32 and running `wrangler dev`

I get this error:

```
workerd/jsg/util.c++:276: error: e = workerd/api/actor-state.c++:794: failed: expected a.getHibernationEventType() != nullptr
```

The [docs say this is in beta](https://developers.cloudflare.com/workers/runtime-apis/durable-objects/#websockets-hibernation-api-beta). Does this mean it does not work?

Do you know when it will/what I can do to enable it locally?

Thanks.

**Comment by MellowYarker** [comment]
Sorry, seems like the docs update hasn't gone out yet. Hibernation doesn't work for `wrangler dev` yet, your best bet at this point is to deploy a worker and use it for testing.

**Comment by threepointone** [comment]
This now works with wrangler dev in newer versions

**Comment by MellowYarker** [comment]
Thanks for the reminder Sunil, fixed by https://github.com/cloudflare/workerd/pull/788 included in release [v1.20230628.0](https://github.com/cloudflare/workerd/releases/tag/v1.20230628.0), which I believe is used in Wrangler 3.1.2 or greater.
